user_study_pilot/tutorial2_heatmap.html

```diff
@@ -1246,6 +1246,12 @@
         id: 'heatmap1',
         title: 'Penguin Species by Island',
         selector: 'g[id^="geom_rect"] > rect',
+        labels: {
+          x: 'Island',
+          y: 'Species',
+          title: 'Penguin Species by Island',
+          fill: 'Count',
```

Comment:
Put the `fill` before `title` in the schema if possible.

```suggestion
          fill: 'Count',
          title: 'Penguin Species by Island',
```